src/devices/Bmxx80/ReadResult/Bmp280ReadResult.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using UnitsNet;
+
+namespace Iot.Device.Bmxx80.ReadResult
+{
+    /// <summary>
+    /// Contains a measurement result of a Bmp280 sensor.
+    /// </summary>
+    public class Bmp280ReadResult
+    {
+        /// <summary>
+        /// Collected temperature measurement. NaN if no measurement was performed.
+        /// </summary>
+        public Temperature Temperature;
```

Comment:
since this is public API this should be a property (`Temperature Temperature { get; set; }`) - here and in all similar places